apis/r/R/utils-coercions.R

```diff
@@ -0,0 +1,40 @@
+#' Coercion methods for SOMA classes
+
+#' @importFrom arrow as_arrow_table
```

Comment:
Should we reexport `as_arrow_table`?
```r
#' @importFrom arrow as_arrow_table
#' @export
#'
arrow::as_arrow_table
```
If not, these methods will be inaccessible to the end-user without `library(arrow)` first